package.json

```diff
@@ -57,7 +57,7 @@
     "@angular/platform-browser": "~12.0.0",
     "@angular/platform-browser-dynamic": "~12.0.0",
     "@angular/router": "~12.0.0",
-    "@cds/core": "^5.0.0",
+    "@cds/core": "5 - 6",
```

Comment:
I don't think a range is a good way to declare a dev dependency.

Reply:
I think we should use `^6.0.0` here as it's more clear and functionally equivalent.